src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -420,17 +423,23 @@ def _encode_prompt(
 
         return prompt_embeds
 
-    def run_safety_checker(self, image, device, dtype):
-        if self.safety_checker is not None:
-            safety_checker_input = self.feature_extractor(self.numpy_to_pil(image), return_tensors="pt").to(device)
+    def run_safety_checker(self, image, device, dtype, output_type="pil"):
```

Comment:
Actually before I think the function `run_safety_checker` returned "np" by default so to be fully backward compatible can we change this here:
```suggestion
    def run_safety_checker(self, image, device, dtype, output_type="np"):
```